README.rst

```diff
@@ -299,6 +299,13 @@ Usage example::
 
     checkcommit --message "$(git log --format=%B -n 1)"
 
+If, for some reason, you wish to skip this QA check for a specific commit message
+you can add ``#noqa`` to the end of your commit message.
+
+Usage example::
+
+    commitcheck --message "[qa] Improved Y #20\n\n Simulation of a special unplanned case\n\n#noqa"
```

Comment:
I'd replace the usage of the command with the commit message text instead, eg:
```
[qa] Improved #20

Simulation of a special unplanned case
#noqa
```

Reply:
👍